configData.py

```diff
@@ -64,17 +64,18 @@ def __init__(self):
         }
 
         self.videoToUse: str = ""
+        self.questionCount: int = 3
+        self.generationModel: str = "BERTopic"
```

Comment:
Nice the default seems to be `BERTTopic` when nothing is specified in the `.env`